core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -59,7 +59,7 @@ public abstract class ViewCatalogTests<C extends ViewCatalog & SupportsNamespace
 
   protected abstract Catalog tableCatalog();
 
-  @TempDir private Path tempDir;
+  @TempDir protected Path tempDir;
```

Comment:
This was needed for TestHiveViewCommits [test](https://github.com/apache/iceberg/pull/9852/files#diff-3613527b53afae01af4bbe834e98a0439b4cf290616d66fc3ea9f28abc2bba03R102)

Reply:
`TestHiveViewCommits` doesn't extend this class, so I don't think this is needed